packages/@sanity/block-tools/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@sanity/block-tools",
+  "version": "0.115.2",
```

Comment:
The version should be synced with the version from `lerna.json`, which is currently at `0.118.5`.

To avoid errors on publish, you should also set access to public in a `publishConfig` section in package.json, e.g.:
```json
  "publishConfig": {
    "access": "public"
  }
```